### DISCUSSION TITLE
mehrere Wechselrichter an einem DTU

### DISCUSSION THREAD

**Discussion by lethuer** [tbnobody/OpenDTU#732] [Q&A]
Ich starte direkt in die vollen und denke über die Anschaffung eines Balkonkraftwerkes nach.
Anstatt eines großen 1500w wechselrichters mit 2 mpp trackern denke ich darüber nach beispielsweise 2x800w wechselrichter zu verwenden, die untereinander verbunden werden. 
beide zusammen würden dann also in dieselbe steckdose einspeisen.
Der Vorteil läge dann in 4 vorhandenen Mpp Trackern, was bei teils verschatteten modulen sicher von Vorteil wäre.

Ich hab noch was von "Multi Inverter Support" gelesen, aber was bedeutet das genau ?
Ist es mit einer einzigen OpenDTU möglich beide Wechselrichter auszulesen / zu steuern ?

**Comment by Solarteur** [comment]
Zitat aus "Features for end users": Default source supports up to 10 inverters.
Bei mir z. B. laufen zwei Inverter und werden von einer OpenDTU ausgelesen. Wichtig ist, dass die Inverter räumlich so angeordnet sind, dass alle auch von einer OpenDTU über Funk erreicht werden können.

**Reply by lethuer** [reply]
Ich stelle mir vor die Leistung je nach aktuellem Verbrauch hochzufahren, sofern denn welche erzeugt wird.
Das stelle ich mir bei zwei Wechselrichtern dann etwas komplizierter vor als bei nur einem ?

**Comment by FrodoVDR** [comment]
> Ich starte direkt in die vollen und denke über die Anschaffung eines Balkonkraftwerkes nach. Anstatt eines großen 1500w wechselrichters mit 2 mpp trackern denke ich darüber nach beispielsweise 2x800w wechselrichter zu verwenden, die untereinander verbunden werden. beide zusammen würden dann also in dieselbe steckdose einspeisen. Der Vorteil läge dann in 4 vorhandenen Mpp Trackern, was bei teils verschatteten modulen sicher von Vorteil wäre.
> 
> Ich hab noch was von "Multi Inverter Support" gelesen, aber was bedeutet das genau ? Ist es mit einer einzigen OpenDTU möglich beide Wechselrichter auszulesen / zu steuern ?

Man kann auch einen HMS-1800T4 oder HMS-2000T4 verwenden beide haben 4 MPP Tracker und sind ebenfalls nur einphasig. Die OpenDTU hierfür wird es offiziell in ein paar Wochen geben, zur Zeit gibt es nur eine Betaversion als Binary, welche aber bereits recht gut funktioniert. Was allerdings etwas problematisch ist das ein anderes Funkmodul benötigt wird welches aktuell nur als SMD verfügbar ist.
[https://discordapp.com/channels/984173303147155506/1082394022359871499](url)

**Reply by lethuer** [reply]
Der Link scheint nicht zu funktionieren.
Ich hab jetzt eine funktionierende dtu mit einer breakout Platine hier liegen… wäre die neue Antenne pin kompatibel ? Zur bisherigen ?

**Comment by harryheizer** [comment]
Hallo Team
ich habe 7 Hoymiles HM-800 - und konnte diese auch alle in der Oberfläche eingeben - aber angezeigt werden aktuell nur die ersten 3 - kann mir einer von Euch helfen ?
habe Firmware 9681ede

**Reply by DPO99** [reply]
Was heißt denn es werden nur die ersten drei angezeigt? Auf der Live-View? Und aktualisiere doch mal auf eine neuere Version. Würde sagen mindestens auf diese [hier](https://github.com/tbnobody/OpenDTU/releases/tag/v23.4.12).

**Reply by harryheizer** [reply]
ja - okay - werde ich probieren - habe aber irgendwie immer Probleme beim flashen... keine Ahnung was ich da immer falsch mache..

**Reply by DPO99** [reply]
> ja - okay - werde ich probieren - habe aber irgendwie immer Probleme beim flashen... keine Ahnung was ich da immer falsch mache..

Du nutzt doch die 23.4.4 oder? Da kannst doch ein OTA Update machen. Musst nicht neu flashen. Ich bilde mir ein, dass die 23.4.4. nach dem 15.3. published wurde.

**Reply by tbnobody** [reply]
Genau genommen wurde die Version 23.4.4 am 4. April 2023 veröffentlicht :)

**Reply by tbnobody** [reply]
werden unter settings --> inverter alle angezeigt? auch mit korrektem typ?

**Comment by harryheizer** [comment]
JaDanke Hat geklappt...
-------- Ursprüngliche Nachricht --------Von: DPO99 ***@***.***> Datum: 18.04.23  20:17  (GMT+01:00) An: tbnobody/OpenDTU ***@***.***> Cc: harryheizer ***@***.***>, Comment ***@***.***> Betreff: Re: [tbnobody/OpenDTU] mehrere Wechselrichter an einem DTU
  (Discussion #732) 

ja - okay - werde ich probieren - habe aber irgendwie immer Probleme beim flashen... keine Ahnung was ich da immer falsch mache..

Du nutzt doch die 23.4.4 oder? Da kannst doch ein OTA Update machen. Musst nicht neu flashen. Ich bilde mir ein, dass die 23.4.4. nach dem 15.3. published wurde.

**Reply by DPO99** [reply]
Siehts Du jetzt auch alle Inverter? Falls ja, gerne die Frage schließen.

**Reply by harryheizer** [reply]
Ja - danke -

**Comment by SMGatAUXde** [comment]
Hätte ebenfalls eine Frage bzgl. 2 WR:

Betreibe aktuell einen HM-1500 an der OpenDTU. Würde nun gerne einen HM-2250 ergänzen. 
In der ReadMe wird aber beschrieben, dass dann ein anderes RF Modul --> CMT2300A nötig ist.

Kann dieses nun bei der bestehenden Hardware zu der NRF24L01+ ergänzt werden, oder muss ich eine zweite OpenDTU bauen und integrieren? 

MfG Sascha

**Reply by DPO99** [reply]
Das sollte gehen, musst aber das PIN Mapping entsprechend anpassen und stecken. Hier mal ein Auszug aus:

https://github.com/tbnobody/OpenDTU/blob/master/docs/DeviceProfiles/nodemcu_esp32.json

[
    {
        "name": "NRF24 + CMT2300A",
        "nrf24": {
            "miso": 19,
            "mosi": 23,
            "clk": 18,
            "irq": 16,
            "en": 4,
            "cs": 5
        },
        "cmt": {
            "clk": 12,
            "sdio": 14,
            "cs": 27,
            "fcs": 26,
            "gpio2": -1,
            "gpio3": -1
        }
    }
]

**Comment by Jens869** [comment]
ich habe einen HM-800 und einen HM-350 dran. Der HM-800 hat zwei Tracker und der HM-350 Einen.
Beide Hoymiles können separate limitiert werden. Also der HM-800 für beide, der HM-350 extra.
OpenDTU zeigt die Gesamteinspeisung an.
